### ISSUE TITLE
Initialize update

### ISSUE THREAD

**Issue by linrio** [anishathalye/neural-style#55]
In _stylize.py: 113_ , changing
```
            sess.run(tf.initialize_all_variables())
```

To
```
            sess.run(tf.global_variables_initializer())
```

**Comment by anishathalye** [comment]
Did the API change or something?

Send me a PR and I'll merge it.

**Comment by avoroshilov** [comment]
Yes, API changed a while ago, deprecating but not yet removing the `tf.initialize_all_variables()` (the warning is quite old), also this - and according to my experiments nothing else - is required to move the project to TF1.0 that have just released

**Comment by anishathalye** [comment]
Fixed in b325725194ac232b50e4744098b726a39062ed4f
